examples/server.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Legionth\React\Http\HttpServer;
+use React\Socket\Server as Socket;
+use React\Promise\Promise;
+use RingCentral\Psr7\Response;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$callback = function() {
+    return new Response();
```

Comment:
I suppose this example would be more useful if there was any kind of content here :)